astro/src/content/docs/extend/events-and-webhooks/events/user-login-failed.mdx

```diff
@@ -47,6 +47,29 @@ export const eventType = 'user.login.failed';
     Moved to <InlineField>event.info</InlineField> in 1.30.0
   </APIField>
 
+  <APIField slot="trailing-fields" name="event.reason" type="Object" since="1.53.0">
+    An object containing data on the reason the login failed.
+  </APIField>
+
+  <APIField slot="trailing-fields" name="event.reason.code" type="String" since="1.53.0">
+    The cause of the event. The possible values are:
+
+    * `credentials` 
+    * `lambdaValidation`
+  </APIField>
+
+  <APIField slot="trailing-fields" name="event.reason.lambdaId" type="UUID" since="1.53.0">
+    This is the unique Id of the lambda that caused the login to fail.
+
+    This field is only present if `reason.code` is `lambdaValidation`.
```

Comment:
We generally use `InlineField` for this use case. 

So it would be something like `This field is only present if `<InlineField>reason.code</InlineField> is `lambdaValidation`. 

The first reference is to an API field, and the second is a reference to a specific value.  Same for `event.reason.lambdaResult`.

Reply:
Updated.